connection/cluster/ClusterServerClient.java

```diff
@@ -52,12 +52,17 @@ class ClusterServerClient extends TypeDBClientImpl {
         this.address = address;
         channel = createManagedChannel(address, credential);
         stub = new ClusterServerStub(channel, credential);
+        isOpen = false;
+    }
+
+    protected void open() {
         try {
             stub.connectionOpen(openReq());
         } catch (Exception e) {
             close();
             throw e;
         }
+        isOpen = true;
     }
```

Comment:
This appears identical with the `CoreClient` implementation and should probably be implemented on `TypeDBClientImpl`.